cli/command/formatter/image.go

```diff
@@ -95,21 +95,23 @@ func imageFormat(ctx ImageContext, images []types.ImageSummary, format func(subC
 			repoDigests := map[string][]string{}
 
 			for _, refString := range append(image.RepoTags) {
```

Comment:
got missed as part of a refactor, thought there was another PR that cleaned it up but don't see it